src/main/webapp/createYourOwnStyle.css

```diff
@@ -44,3 +51,60 @@ nav a {
     padding-right: 35px;
     font-family: 'Roboto', sans-serif;
 }
+
+#new-initiative {
+    background-color: #F6E1E6;
+    border-radius: 25px;
+    width: 900px;
+    margin: 0 auto;
+}
+
+#newInitiativeForm {
+    text-align: left;
+    padding-left: 140px;
+    padding-right: 110px;
+    padding-top: 40px;
+    padding-bottom: 40px;
+}
+
+.initiative-input {
+    font-family: 'Roboto', sans-serif;
+    color: black;
+}
+
+input[type=text] {
+    border: 3px solid #FFFDFD;
+    background-color: #F6E1E6;
+    border-radius: 12px;
+    height: 27px;
+    color: #847C7C;
+    font-family: 'Roboto', sans-serif;
+}
+
+input[type=file] {
+    font-family: 'Roboto', sans-serif;
+}
```

Comment:
If you change some of the types for the input tags, you could consider putting all those input tags in a div and just apply these styling to the div.

Reply:
I think text and file look fine, but I will work on fixing the date tag ones.